src/components/pages/ImageClassification/SampleUnitInputSelector.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { ButtonPrimary, ButtonSecondary } from '../../generic/buttons'
+import { H3 } from '../../generic/text'
+import { IconSparkles, IconPen } from '../../icons'
+import styled from 'styled-components/macro'
+import language from '../../../language'
+import theme from '../../../theme'
+import PropTypes from 'prop-types'
+
+const ButtonText = styled.span`
+  margin-left: 0.8rem;
+`
+
+const ButtonContainer = styled.div`
+  margin-bottom: 1rem;
+`
+
+const SelectorContainer = styled.div`
+  padding: 1rem;
+  background-color: ${theme.color.grey5};
+  margin-left: 1rem;
+  margin-top: 3rem;
+`
+
+const TextContainer = styled.div`
+  max-width: 68rem;
+`
+
+const SampleUnitInputSelector = ({ setObservationTableType }) => {
+  const handleSampleUnitChange = (type) => () => {
+    setObservationTableType(type)
+  }
+
+  return (
+    <SelectorContainer>
+      <TextContainer>
+        <H3 htmlFor="image-classification-selection">
+          {language.imageClassification.sampleUnitInputSelector.title}
```

Comment:
You could potential destructure the relevant object in the language file to make this less verbose.
e.g.
```const { sampleUnitInputSelector: sampleUnitInputSelectorLanguage } = language.imageClassification```
or
```const { title, description, etc} = language.imageClassification.sampleUnitInputSelector```

I did something similar in the GFCR components: https://github.com/data-mermaid/mermaid-webapp/blob/d0a33c14bf8b8122d8ff6483f5db6c7db284486f/src/components/pages/gfcrPages/GfcrIndicatorSetForm/subPages/F1Form.js#L11

Reply:
good idea - will update this in the next branch and also use this in other components 🙌